src/components/ProjectsSection/ProjectsSection.js

```diff
@@ -24,46 +23,85 @@ const ProjectsSectionItemWrapper = styled(SectionItemWrapper)`
-  max-width: ${rem(150)};
+const StyledImg = styled(Img)`
+  width: 90%;
+
+  @media (min-width: ${breakPoints.largeMobile}) {
+    width: 60%;
+  }
 
   @media (min-width: ${breakPoints.desktop}) {
-    width: 30%;
-    max-width: none;
-    height: unset;
+    width: 40%;
   }
 `;
 
 const ProjectsSectionItemText = styled.p`
+  text-align: center;
+
   @media (min-width: ${breakPoints.desktop}) {
-    width: 70%;
+    width: 60%;
+    padding: 0 ${rem(20)};
     text-align: ${({ isReversed }) => (isReversed ? 'right' : 'left')};
   }
 `;
 
-const ProjectsSection = () => (
-  <SectionWrapper id="projects">
-    <SectionHeader>
-      <FormattedMessage id="header.projectsSection" />
-    </SectionHeader>
-    <ProjectsSectionItemWrapper>
-      <ProjectsSectionItemIcon>
-        <Image />
-      </ProjectsSectionItemIcon>
-      <ProjectsSectionItemText>
-        <FormattedMessage id="description.projectsSection.hangman" />
-      </ProjectsSectionItemText>
-    </ProjectsSectionItemWrapper>
-    <ProjectsSectionItemWrapper isReversed>
-      <ProjectsSectionItemIcon>
-        <Image />
-      </ProjectsSectionItemIcon>
-      <ProjectsSectionItemText isReversed>
-        <FormattedMessage id="description.projectsSection.schoolApp" />
-      </ProjectsSectionItemText>
-    </ProjectsSectionItemWrapper>
-  </SectionWrapper>
-);
+const ProjectsSection = () => {
+  const images = useStaticQuery(graphql`
+    query {
+      hangman: file(relativePath: { eq: "screenshots/hangman-desktop-1.png" }) {
+        childImageSharp {
+          fluid {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+      schoolApp: file(
+        relativePath: { eq: "screenshots/schoolapp-desktop-0.png" }
+      ) {
+        childImageSharp {
+          fluid {
+            ...GatsbyImageSharpFluid
+          }
+        }
+      }
+    }
+  `);
+  return (
+    <SectionWrapper id="projects">
+      <SectionHeader>
+        <FormattedMessage id="header.projectsSection" />
+      </SectionHeader>
+      <ProjectsSectionItemWrapper>
+        <StyledImg fluid={images.hangman.childImageSharp.fluid} />
+
+        <ProjectsSectionItemText>
+          <a
+            href="https://charliemantium.github.io/hangman/"
+            target="_blank"
+            rel="noopener noreferrer"
+            display="block"
+          >
+            Hangman
+          </a>
+          <FormattedMessage id="description.projectsSection.hangman" />
+        </ProjectsSectionItemText>
+      </ProjectsSectionItemWrapper>
+      <ProjectsSectionItemWrapper isReversed>
+        <StyledImg fluid={images.schoolApp.childImageSharp.fluid} />
+        <ProjectsSectionItemText isReversed>
+          <a
+            href="https://mantium-school-app.herokuapp.com/"
+            target="_blank"
+            rel="noopener noreferrer"
+            display="block"
+          >
+            SchoolApp
+          </a>
+          <FormattedMessage id="description.projectsSection.schoolApp" />
```

Comment:
you should use translations and inject `School App`. Check this one:
https://github.com/formatjs/react-intl/blob/master/docs/Components.md#rich-text-formatting